extensions/spatial/src/org/sbml/jsbml/validator/offline/constraints/InteriorPointConstraints.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * ----------------------------------------------------------------------------
+ * This file is part of JSBML. Please visit <http://sbml.org/Software/JSBML>
+ * for the latest version of JSBML and more information about SBML.
+ * 
+ * Copyright (C) 2009-2018 jointly by the following organizations:
```

Comment:
Just noticed the header you where using, you should replace 2018 by 2019 in all your files.
And you need to remove "* 4. The University of California, San Diego, La Jolla, CA, USA" and put the Babraham as number 4.

Reply:
Done!